compound/build.gradle.kts

```diff
@@ -69,20 +69,19 @@ android {
     }
 
     dependencies {
+        implementation(libs.androidx.activity.activity)
         implementation(libs.androidx.compose.material3)
         implementation(platform(libs.androidx.compose.bom))
         implementation(libs.ui.tooling.preview.android)
         implementation(libs.kotlinx.collections)
 
+        // Showkase
         implementation(libs.showkase)
         ksp(libs.showkase.processor)
         kspTest(libs.showkase.processor)
 
-        implementation(libs.accompanist.systemui)
```

Comment:
I think accompanist is not used anymore and so can be removed from the file libs.versions.toml?